src/nav-component/Navigation.js

```diff
@@ -0,0 +1,113 @@
+import { routeNames } from "../misc/Constants";
+import { useCartAndWishlist } from "../context/CartAndWishlistProvider";
+import { NavLink } from "react-router-dom";
+import styles from "./Navigation.module.css";
+import { NavigationAside } from "../nav-aside-component/NavigationAside";
+import { useState } from "react";
+import { useAuth } from "../context/AuthProvider";
+
+export const Navigation = ({}) => {
+  const { cartAndWishlistState } = useCartAndWishlist();
+  const [isAsideOpen, setIsAsideOpen] = useState(false);
+  const{isUserLoggedIn}=useAuth();
+
+  const cartLength = cartAndWishlistState.cart.filter(
+    (item) => !item.wishlisted
+  ).length;
+  const wishlistLength = cartAndWishlistState.cart.filter(
+    (item) => item.wishlisted
+  ).length;
+
+  return (
+    <>
+      <NavigationAside
+        isAsideOpen={isAsideOpen}
+        setIsAsideOpen={setIsAsideOpen}
+      />
+      <div className={styles.header}>
+        <div className={styles.logo}>
+          <img src="../../assets/logo.png" alt=""></img>
+        </div>
+        <nav className={styles.heading}>
+          <ul className="nav">
+            <li className="nav-item">
+              
+              <NavLink to="/" className={styles.navPills}>
+                <svg width="1em" height="1em" viewBox="0 0 24 24">
```

Comment:
SVG can be imported here and in other places as well.

Reply:
have to use img tag instead of svg and have to download all svgs and keep in a folder, will do this later